server/src/DBTablesMonitoring.cc

```diff
@@ -1198,6 +1198,15 @@ string TriggersQueryOption::getCondition(void) const
 			MONITORING_SERVER_SELF_ID);
 	}
 
+	if (m_impl->triggerCollectHost >= REAL_HOST_TRIGGER) {
+		condition += " AND ";
```

Comment:
Thank you for comment.
In order to prevent duplicate of "AND", I use the DataQueryOption::addCondition.
Also in the same way before the code.